### PR TITLE
Rails 7

### PR DESCRIPTION
**Pull request by KludgeKML**
Update to Rails 7 as here:

https://trello.com/c/H1qzRFrl/1204-rails-7-update-search-admin

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -50,32 +53,15 @@
...
-  # Adds additional error checking when serving assets at runtime.
-  # Checks for improperly declared sprockets dependencies.
-  # Raises helpful error messages.
-  config.assets.raise_runtime_errors = true
-
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
-  # Allow requests for all domains e.g. <app>.dev.gov.uk
-  config.hosts.clear
```
Comment (1pretz1) [inline config/environments/development.rb:67]: Think we still need this for govuk-docker

**Review by 1pretz1 — COMMENTED** [review]
Looks good, just one minor comment

File: config/environments/development.rb (hunk 1 above)
Comment (KludgeKML) [reply config/environments/development.rb:67]: Good catch!

Comment (KludgeKML) [reply config/environments/development.rb:67]: Have moved to the bottom, where the config hosts stuff usually is.

**Review by 1pretz1 — APPROVED** [review]

**Pull request merged**